ecl/eclagent/eclagent.cpp

```diff
@@ -3656,14 +3656,7 @@ extern int HTHOR_API eclagent_main(int argc, const char *argv[], StringBuffer *
 #ifdef MONITOR_ECLAGENT_STATUS
         std::unique_ptr<CSDSServerStatus> serverstatus;
 #endif
-        Owned<ILocalWorkUnit> standAloneWorkUnit;
-        if (wuXML)
-        {
-            //Create workunit from XML
-            standAloneWorkUnit.setown(createLocalWorkUnit(wuXML->str()));
-            wuXML->kill();  // free up text as soon as possible.
-        }
-
+        Owned<ILocalWorkUnit> standAloneWorkUnit(std::move(localWu));
```

Comment:
Not sure why this copy is needed rather than using the one passed in

Reply:
I can't think of a good reason either - possibly something to do with the lifetime, but I'll change and simplify